### ISSUE TITLE
Adaptive immune response

### ISSUE THREAD

**Issue by RLovering** [geneontology/go-ontology#18458]
Discussion from https://github.com/geneontology/go-ontology/issues/18370

**Comment by RLovering** [comment]
Ruth's Comment 1:
I have just been looking at this part of the ontology with a clinician with an interest in the innate inflammatory response, in particular IL6.
My arguement, like Alex's has always been that inflammation occurs without an infection or toxins, however the problem is that the innate immune cells are involved in inflammation, and that the body does have to detect a change in the tissue or cells that leads to an inflammatory response. So we were discussing if an immune response always leads to inflammation, the exception might be vaccination, but this is often associated with aching joints or raised temperate.

So I am now sitting on the fence on this one, but if these were to be linked in any way I would make inflammation the part_of parent to the immune response, because not every occurance of inflammation is an immune response, whereas I think an immune response leads to some sort of inflammation.

Finally I am very concerned about the current ontology. Why isn't immune response a child of defense response? and why is innate immune response a child of GO:0098542 defense response to other organism
Screenshot 2019-12-12 at 15 09 00


I made this suggestion for 2 reasons:

    currently adaptive immune response is not a child of defense response and curators do not realise this and therefore there is inconsistent annotation of proteins involved in adaptive immunity:
    approx 250 human proteins associated with adaptive immune response AND defense response
    150 proteins (not Ig or TCR or HLA)
    74 Immunoglobulin (ig)
    13 T-cell receptor (TCR)
    16 HLA

approx 320 human proteins ONLY associated with adaptive immune response (ie NOT associated with defense response)
140 proteins (not Ig or TCR or HLA)
87 IG
113 TCR
4 HLA

So I thought we need a term that will enable a curator to annotate to both adaptive immune response AND defense response without having to create 2 annotations.

    The innate immune response term has child terms (see figure below), which look like they were created this year, or maybe just were edited recently, and I thought it made sense to have the equivalent terms in the adaptive immune response.

Although now you mention it you are probably right that, as the adaptive immune response is adaptive, perhaps none of the proteins are organism/antigen specific, which they are in the innate immune response. I just thought that some of the regulators of the adaptive immune response might be organism/antigen specific.

Maybe we could just have adaptive immune response involved in defense response?

Alex comment 1:

'immune response' is not a child of 'defense response to other organism' because regulatory immune responses are not defense responses to other organisms and autoimmune responses are not defense responses to other organisms.

Note that not all autoimmune responses are 'abnormal'. Some occur as regulatory immune responses to suppress other autoimmune responses.

Ruth Comment 2:
Good point but there should be an adaptive immune response child term that has the parent defense response to other organism. You can't expect curators to realise that in order to be consistant in the annotation of innate immune response genes and adaptive immune response genes they have to add the additional defense response annotation
I have just used QuickGO to look at the reviewed human proteins associated with adaptive immune response and identified 600 proteins associated with this term (obviously lots of IG and TCR genes included). Of these 600 proteins:

Also associated with defense response
150 proteins (not Ig or TCR or HLA)
74 IG
13 TCR
16 HLA

Only associated with adaptive immune response (ie NOT associated with defense response)
140 proteins (not Ig or TCR or HLA)
87 IG
113 TCR
4 HLA

I think a GO term is needed along the lines of GO:0090720 primary adaptive immune response, and GO:0140367 antibacterial innate immune response etc. ie
antibacterial adaptive immune response
Then there could be the parent:
antibacterial immune response
which would also have the child term: GO:0032496 response to lipopolysaccharide
Perhaps we could identify all proteins associate with both
GO:0032496 response to lipopolysaccharide and either adaptive immune response OR innate immune response and annotate them to the relevant antibacterial immune response term.

GO:0042742 defense response to bacterium

    is_a child NEW GO term: antibacterial immune response

        is_a child GO:0140367 antibacterial innate immune response
        is_a child GO:0032496 response to lipopolysaccharide
        is_a child NEW GO term: antibacterial adaptive immune response

And repeat as above for virus and fungi.

[DefenceResponsevAdaptiveResponse.xlsx](https://github.com/geneontology/go-ontology/files/3967628/DefenceResponsevAdaptiveResponse.xlsx)

Alex Comment 2
It would be great to see the analysis (email it directly or attach it here).

I can see the value in terms like 'antibacterial adaptive immune response' as a way of have a child of both adaptive immune response and defense response, particularly as a way of simplifying the annotation process. I think that one reason I have not previously seen the need for this type of defined class is that any gene product that that is annotated to adaptive immune response could in theory be used for any subtype of adaptive immune response to particular organisms, that the whole point of being adaptive is that in theory antigens from any source can be targeted, and a memory enhanced immune response will be produced.

Of course there are things like NK T cells and some classes of gamma-delta T cells that have TCRs preselected for antigens of bacterial origin because of the use of particular V gene segments. But arguably, the immune response driven by these cells are really innate rather than adaptive, or perhaps innate with processes typically associated with adaptive immune memory occurring in the T cells.

TL;DR. Terms like 'antibacterial adaptive immune response' are okay with me.

Val Comment 1

Do we really need to encode the organism in the immune response terms?

In reviewing the ontology and the existing annotations, most of the annotation inconsistencies are due to some curators annotating to terms under "innate immune response" and some to "defence response to fungus".

It would require A LOT of duplication to create

to cover the existing
response to fungus
response to protazoan
response to bacterium
response to insect.

The "other organism" can be captured by the taxon?

Curators are finding the high-level "response to x " terms, but not the terms under "innate immune response"

So for example for plants, the pathway activated is often

GO:0009626 plant-type hypersensitive response
we would not really want to have

plant-type hypersensitive response in response to fungus
plant-type hypersensitive response in response to virus
plant-type hypersensitive response in response to oomycete

It would be much better to concentrate on pathway and mechanism, and take species out of the equation (or capture species in and other way)

One alternative would be to allow "in response to fungus/bacteria" as an extension, in much the same way as we can have "phases" in extensions now.

Alex Comment 3
Clearly, the original approach avoided the duplication of hierarchies Val warns of, but on the other hand, Ruth and her group do annotation in this area and point out the advantages of additional pre-composed terms. I don't have a strong opinion either way, though I continue to see the value of pre-composed terms as seemingly a faster approach to annotation than complex GO-CAM assemblages. But I don't currently do annotation so perhaps my sense of the relative difficulty of building up GO-CAM representations from simpler classes vs using "traditional" precomposed terms is not supported by practice.

Val comment 2
All GO curators should be familiar with annotation extensions and their use - we have been using them since before 2014, so this shouldn't be an issue.

One thing for sure is that the current situation results in suboptimal and inconsistent annotation.

Many people are annotating only to the "response to fungus" terms which do not have children, rather than biologically informative terms. I recoreded and example here:

#18072

By follwing the "response to fungus" branch my collaborator annotated to
GO:0038187 | pattern recognition receptor activity | IMP | B | part_of cell surface pattern recognition receptor signaling pathway , part_of response to fungus

GO:1900150 | regulation of defense response to fungus
(which has no descendants)
For the same annotation, I did

GO:0038187 | pattern recognition receptor activity | IMP | B | part_of cell surface pattern recognition receptor signaling pathway , part_of response to fungus

Extensions should not be a problem as it is easy to configure the tools to suggest the appropriate extension types for specific terms.

However, my preferred solution would be to get rid of the "response to species" and concentrate on the biological mechanism. I can see why this might be a step too far though...but we definitely need to do something.

Alex comment 4
I always like to think of the value of GO terms and their annotations in how they show up in term enrichment results (or ideally would do so if annotation were more complete). The challenge with annotation extensions is that term enrichment algorithms do not consider or show these things in results, and thus they have no value to 98% of the consumers of GO. Ideally we need better tools for displaying and analyzing the full range of information we collect as part of the GO annotation process, and I am not arguing we should not make complex GO annotations. But if we remove terms such as the response to species ones, we remove the ability to collect valuable contextual information in the annotation process contained in these term names and definitions that will appear in (and has always appeared in) term enrichment results and is surely of value to our users.
@ValWood

Val comment 3

But "response to fungus" is not as informative in an enrichment than a biological process or pathway. Consistency improves enrichments and currently, the useful terms are really hidden by these "response to organism" branches. I would much rather have real processes and signalling pathways in my enrichment, and then figure out if it was relevant to my pathogen of interest (which presumably provided the genes in my experimental input ? so it isn't really telling me anything new?).

Plus I would now need to request "response to fungal specific terms for all of the processes I have annotated, linked to the "response to fungus" terms and move my annotations down to them, even if the editors agree to do this.

Extensions provide a useful way to capture this information which is slightly orthogonal, and hopefully, by the time there are enough annotations to be useful the tools will have caught up
and allow enrichment over extensions.

**Comment by RLovering** [comment]
Val,
not all groups are currently using AEs.
This domain needs some sort of revisions so that we improve annotation consistency and capture specific roles of different proteins.
Perhaps as a start I should just IC all annotations to adaptive immune response to defense response for now.
Ruth

**Comment by pgaudet** [comment]
@addiehl

**Comment by RLovering** [comment]
There is no way I can look at this until 2023

**Comment by ValWood** [comment]
It's OK you only got a message because I tried to remove myself as an assignee, removed everyone and then added you all back.
